.github/.kodiak.toml

```diff
@@ -0,0 +1,4 @@
+version = 1
+merge.blocking_labels = ["WIP", "blocked"]
+merge.notify_on_conflict = false
+approve.auto_approve_usernames = ["renovate"]
```

Comment:
PRs from these names will be approved by kodiak
this means you can batch-assign `automerge` in the PR overview for these PRs to merge together.